packages/get-platform/src/getPlatform.ts

```diff
@@ -12,6 +12,7 @@ const exec = promisify(cp.exec)
 
 // https://www.geeksforgeeks.org/node-js-process-arch-property/
 export type Arch = 'x32' | 'x64' | 'arm' | 'arm64' | 's390' | 's390x' | 'mipsel' | 'ia32' | 'mips' | 'ppc' | 'ppc64'
+export type UnameArch = 'i386' | 'x86_64' | 'arm64' | 'arm7l'
```

Comment:
```suggestion
export type UnameArch = 'i386' | 'x86_64' | 'arm64' | 'arm7l' | 'aarch64'
```

`arm64` is on macOS (which we might not even care about here), on Linux `uname -m` will report `aarch64`

Reply:
There's also quite a bunch of other variations only of these architectures that this type doesn't cover, not even counting completely different architectures covered by the `Arch` type. Do we need to enumerate them (i.e. are you planning to match on this type anywhere), or could we just use `string` instead?